config.base.yaml

```diff
@@ -28,6 +35,7 @@ audits:
       - mockevent
     alerts:
       - filestore
+      - slackalert
```

Comment:
Please remove `slackalert` from the audit section. The `slackalert` definition at line 20 above should remain. Only this one at line 38 should be removed. The reason is that after we clone `cloudmarker` from GitHub repo, it should run without any errors without any further configuration. Since `slackalert` requires additional configuration, it should be added by the user only if they want to do so.